mods/ra/weapons/ballistics.yaml

```diff
@@ -49,96 +26,57 @@
 		ImpactSounds: splash9.aud
 		ValidImpactTypes: Water
 
-105mm:
-	ReloadDelay: 70
-	Range: 4c768
-	Report: cannon1.aud
-	Burst: 2
-	BurstDelay: 4
+25mm:
+	Inherits: ^Cannon
```

Comment:
This adds explosion sounds that didn't exist before.  I know from experience that some people might consider this fix to be an annoyance/regression, but I don't have a strong opinion either way.